### ISSUE TITLE
Shadowed buttons remain optically pressed

### ISSUE THREAD

**Issue by netfaresi** [olton/Metro-UI-CSS#1643]
**Describe the bug**
When clicked on shadowed buttons, they keep their pressed status optically.

**Expected behavior**
They should have shadow after click.

**Desktop (please complete the following information):**
 - OS: Windwos 10
 - Chrome, Firefox, MS Edge, MSIE, Opera
 - Metro Version 4.4.0 (also the previous version)

**Smartphone (please complete the following information):**
 - Device: Samsung S10
 - OS: Android
 - Browser Chrome Mobile
 - Metro Version: used in official Metro-UI homepage

**Comment by olton** [comment]
how do you define a button with a shadow?

**Comment by netfaresi** [comment]
Hello Serhii,

thank you for your feedback. This is the shadowed button which I used: 

**`<button id="saveDocumentType"  type="button" class="button drop-shadow">
       <span class="mif-floppy-disk"></span> Save document
</button>`**

I analyzed the issue myself and found that the source of the bug (if we should it name bug?) are the pseudo classes active, focus etc., which override some properties of the button, so that it looks like pressed even after having released the mouse button.

Just a couple minutes ago, I solved my problem via defining following event handler:

 **`$("button.drop-shadow").mouseup(function(){$(this).blur();});`**

It works, but it is not an elegant solution I think.

Best Regards
